guides/faq.rst

```diff
@@ -160,7 +160,7 @@ If you find some, please do however report them.
 
 To install the dev version of ESPHome:
 
-- In Home Assistant: Add the ESPHome repository `https://github.com/esphome/hassio <https://github.com/esphome/hassio>`
+- In Home Assistant: Add the ESPHome repository `https://github.com/esphome/hassio <https://github.com/esphome/hassio>`__
```

Comment:
IMHO this should not be a link at all, as it should be copy-pasted, not visited.

Reply:
I considered that too.  right-click copy vs. select and copy?  Either way is better than what it was.

Reply:
I second this

Reply:
Honestly, if you do that, than at least adjust the readme on the target link to provide instruction.

Even better, this whole thing can be replaced with a link that directly adds it to HA.

Reply:
Maybe we can get a fancy click to copy link?

Reply:
More links and buttons: <https://my.home-assistant.io/create-link/>

Reply:
ahh sorry I didnt expect a conversation to develop and as it was looking ugly: 

![image](https://user-images.githubusercontent.com/96162/124183501-b150b600-da8e-11eb-8f48-ad8a118cf0b8.png)

I just merged it 😀